FluentTerminal.App/App.xaml.cs

```diff
@@ -160,9 +160,72 @@ protected override void OnActivated(IActivatedEventArgs args)
+                        {
+                            var exportFile = await ApplicationData.Current.LocalFolder.CreateFileAsync("config.json", CreationCollisionOption.OpenIfExists);
+                            var stream = await exportFile.OpenStreamForWriteAsync();
+                            JsonSerializer s = new JsonSerializer();
+                            using (StreamWriter sw = new StreamWriter(stream))
+                            using (JsonWriter writer = new JsonTextWriter(sw))
+                            {
+                                var config = new
+                                {
+                                    App = _settingsService.GetApplicationSettings(),
+                                    KeyBindings = _settingsService.GetCommandKeyBindings(),
+                                    Themes = _settingsService.GetThemes(),
+                                    Profiles = _settingsService.GetShellProfiles(),
+                                    TerminalOptions = _settingsService.GetTerminalOptions(),
+                                    TabThemes = _settingsService.GetTabThemes()
+                                };
+                                s.Serialize(writer, config);
+                            }
+                            stream.Dispose();
+                        }
+
+                        if (settingsVerb.Import)
+                        {
+                            var stream = await ApplicationData.Current.LocalFolder.OpenStreamForReadAsync("config.json");
+                            JsonSerializer s = new JsonSerializer();
+                            using (StreamReader sr = new StreamReader(stream))
+                            using (JsonReader reader = new JsonTextReader(sr))
+                            {
+                                var config = new
+                                {
+                                    App = new ApplicationSettings(),
+                                    KeyBindings = new Dictionary<string, ICollection<KeyBinding>>(),
+                                    Themes = new List<TerminalTheme>(),
+                                    Profiles = new List<ShellProfile>(),
+                                    TerminalOptions = new TerminalOptions()
+                                };
+                                s.Populate(reader, config);
+                                _settingsService.SaveApplicationSettings(config.App);
+                                
+                                foreach (var pair in config.KeyBindings)
+                                {
+                                    _settingsService.SaveKeyBindings(pair.Key, pair.Value);
+                                }
+
+                                foreach (var theme in config.Themes)
+                                {
+                                    _settingsService.SaveTheme(theme, _settingsService.GetTheme(theme.Id).Equals(default(TerminalTheme)));

```

Comment:
If we only export non preinstalled themes/profiles we can hardcore IsNew to true